app/controllers/declaration/CommodityMeasureController.scala

```diff
@@ -41,20 +41,21 @@ class CommodityMeasureController @Inject()(
 )(implicit ec: ExecutionContext)
     extends FrontendController(mcc) with I18nSupport with ModelCacheable with SubmissionErrors {
 
-  private def form()(implicit request: JourneyRequest[_]) = CommodityMeasure.form(request.declarationType).withSubmissionErrors()
-
   def displayPage(mode: Mode, itemId: String): Action[AnyContent] = (authenticate andThen journeyType) { implicit request =>
+    val commodityCode = request.cacheModel.itemBy(itemId).flatMap(_.commodityDetails).flatMap(_.combinedNomenclatureCode)
```

Comment:
Cannot use `getCommodityCode(itemId)` def ?

Reply:
unused val, must of missed removing it :+1: